### DISCUSSION TITLE
Installer Not Updating

### DISCUSSION THREAD

**Discussion by MjrIncident** [flybywiresim/aircraft#5339] [General]
FYI

https://api.flybywiresim.com/installer - CDN not working

Similarly - https://api.flybywiresim.com/api/v1/download?url=https://flybywiresim-packages.b-cdn.net/vmaster/A32NX-master.zip

Installer is reading "error" so cannot update any version atm - or the installer itself.

![image](https://user-images.githubusercontent.com/72098504/126042476-6716c944-a036-4934-b897-710e61fd8a44.png)
![image](https://user-images.githubusercontent.com/72098504/126042512-e3d81374-f540-4e61-a8d3-f2ffcd92055f.png)

**Comment by Inaxair** [comment]
Hello,

has the issue been solved yet? Also, can you try without an anti-virus or with the firewall desactivated for the duration of the installation?

Best regards,
Inaxair
